download/upgradeRecipe/upgradeRecipe8.adoc

```diff
@@ -17,25 +17,25 @@ Any backwards incompatible changes are annotated with a [.label.label-danger.lab
 
 == Backwards incompatible changes to the public API since 7.x
 
-//[.upgrade-recipe-major.upgrade-recipe-public-api]
-//=== Java 11 or higher required
-//
-//If you're using JRE or JDK 8, upgrade to JDK 11 or higher.
-//
-//JDK 11 or higher is still available for _free_ (including security and bug fixes),
-//alongside Oracle's paid subscription.
-//
-//* On linux, get OpenJDK from your linux software repository.
-//For example on Fedora and RHEL:
-//+
-//[source, bash]
-//----
-//sudo dnf install java-11-openjdk-devel
-//----
-//
-//* On Windows and Mac, https://adoptopenjdk.net[download OpenJDK from AdoptOpenJDK].
-//
-//We currently intend to support a minimal version of Java 11 throughout the entire 8.x series.
+[.upgrade-recipe-major.upgrade-recipe-public-api]
+=== Java 11 or higher required
+
+If you're using JRE or JDK 8, upgrade to JDK 11 or higher.
+
+JDK 11 or higher is still available for _free_ (including security and bug fixes),
+alongside Oracle's paid subscription.
+
+* On linux, get OpenJDK from your linux software repository.
+For example on Fedora and RHEL:
++
+[source, bash]
+----
+sudo dnf install java-11-openjdk-devel
+----
+
+* On Windows and Mac, https://adoptopenjdk.net[download OpenJDK from AdoptOpenJDK].
+
+We currently intend to support a minimal version of Java 11 throughout the entire 8.x series.
```

Comment:
I'd remove this line. In general, let's not make promises years into an uncertain future.
Java LTS are not forever, anymore.

Reply:
We wrote a similar line in the 7.x upgrade recipe and it never was a problem.
The "currently intend" part is to make it clear it's not a solid promise.
Either way, I strongly believe we should not change the target version of the class files we produce in a minor or hotfix version. Which is basically what this is about.

I'd keep this sentence in, but I don't feel strongly about it. Radovan is the tie breaker I guess.

Reply:
Neither do I understand the "We currently intend" as a promise.